### ISSUE TITLE
Standardize return types

### ISSUE THREAD

**Issue by agerlach** [SciML/DiffEqUncertainty.jl#24]
- `expectation(Koopman())` returns a `QuadratureSolution`
- `expectation(MonteCarlo())` returns a number/vector
- `centralmoment` returns a vector

**Comment by ChrisRackauckas** [comment]
They should all be a QuadratureSolution IMO

**Comment by agerlach** [comment]
Agreed. It would be good to try to set resid for all methods too. I know this expensive for MC, so maybe a kwarg flag to turn it on.

**Comment by ChrisRackauckas** [comment]
Yeah, I think by default have nothing there and a flag to turn it on is good.
On 8/18/2020 8:48:32 AM, agerlach <notifications@github.com> wrote:
Agreed. It would be good to try to set resid for all methods too. I know this expensive for MC, so maybe a kwarg flag to turn it on.
